OKTA-503093 updating developer docs for loading page API changes (#3458)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/brands/index.md

Passing an invalid `brandId` returns a `404 Not Found` status code with error code `E0000007`.

> **Note**: `loadingPageTouchPointVariant` returned in response body only if LOADING_PAGE feature flag enabled

Reviewer: ```suggestion
> **Note:** The `loadingPageTouchPointVariant` property is returned in the response body only if the **Loading Page Animation** feature is enabled. <ApiLifecycle access="ea" />
```

Passing an invalid `brandId` or an invalid `themeId` returns a `404 Not Found` status code with error code `E0000007`.

> > **Note:** The response body includes the `loadingPageTouchPointVariant` property only if the LOADING_PAGE feature is enabled.

Reviewer: ```suggestion
> **Note:** The response body includes the `loadingPageTouchPointVariant` property only if the the **Loading Page Animation** feature is enabled. <ApiLifecycle access="ea" />
```

Returns an updated [Theme Response](#theme-response-object)

* Passing an invalid `brandId` or an invalid `themeId` returns a `404 Not Found` status code with error code `E0000007`.

* Passing invalid body parameters returns a `400 Bad Request` status code with error code `E0000001`.

> * Passing the optional `loadingPageTouchPointVariant` body parameter without having the LOADING_PAGE feature enabled returns a `401 Unauthorized` with error code `E0000015`.

Reviewer: ```suggestion
* Passing the optional `loadingPageTouchPointVariant` body parameter without having the the **Loading Page Animation** feature enabled returns a `401 Unauthorized` with error code `E0000015`. <ApiLifecycle access="ea" />
```

| `endUserDashboardTouchPointVariant`   | Enum     | Variant for the Okta End-User Dashboard. Accepted values: `OKTA_DEFAULT`, `WHITE_LOGO_BACKGROUND`, `FULL_THEME`, `LOGO_ON_FULL_WHITE_BACKGROUND`.                    | `OKTA_DEFAULT`    |
| `errorPageTouchPointVariant`          | Enum     | Variant for the error page. Accepted values: `OKTA_DEFAULT`, `BACKGROUND_SECONDARY_COLOR`, `BACKGROUND_IMAGE`.                 | `OKTA_DEFAULT`    |
| `emailTemplateTouchPointVariant`      | Enum     | Variant for email templates. Accepted values: `OKTA_DEFAULT`, `FULL_THEME`.                                                | `OKTA_DEFAULT`    |

> | `loadingPageTouchPointVariant`        | Enum     | (Optional) Variant for the Okta loading page. Applicable if `LOADING_PAGE` feature enabled. Accepted values: `OKTA_DEFAULT`, `NONE`.                                 | `OKTA_DEFAULT`    |

Reviewer: ```suggestion
| `loadingPageTouchPointVariant` <ApiLifecycle access="ea" />       | Enum     | (Optional) Variant for the Okta loading page. Applicable only if the **Loading Page Animation** feature is enabled. Accepted values: `OKTA_DEFAULT`, `NONE`.                                 | `OKTA_DEFAULT`    |
```

| `OKTA_DEFAULT`                  | Use the Okta logo and the Okta colors in the email templates.    |
| `FULL_THEME`                    | Use the logo from Theme and `primaryColorHex` as the background color for buttons.      |

> <ApiLifecycle access="ea" />

Reviewer: ```suggestion
```

> > **Note:** Okta loading page variants are gated behind the **Loading Page Animation** Early Access feature and must be enabled. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features).

Reviewer: ```suggestion
```

> #### Variants for the Okta loading page:

Reviewer: ```suggestion
#### Variants for the Okta loading page:

<ApiLifecycle access="ea" />

> **Note:** Okta loading page variants are only available if the **Loading Page Animation** Early Access feature is enabled. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features).
```
Add the EA flag under the section heading since the feature applies to that section, not the previous one.

| `FULL_THEME`                    | Use the logo from Theme and `primaryColorHex` as the background color for buttons.      |

> #### Variants for the Okta loading page

Reviewer 2: ```suggestion
#### Variants for the Okta loading page:
```

> #### Variants for the Okta loading page:

Reviewer 2: ```suggestion
#### Variants for the Okta loading page
```

| `primaryColorContrastHex`             | String                  | Primary color contrast hex code             |
| `secondaryColorHex`                   | String                  | Secondary color hex code                    |
| `secondaryColorContrastHex`           | String                  | Secondary color contrast hex code           |
| `signInPageTouchPointVariant`         | Enum                    | Variant for the Okta Sign-In Page           |
| `endUserDashboardTouchPointVariant`   | Enum                    | Variant for the Okta End-User Dashboard     |
| `errorPageTouchPointVariant`          | Enum                    | Variant for the error page                  |
| `emailTemplateTouchPointVariant`      | Enum                    | Variant for email templates                 |

> | `loadingPageTouchPointVariant`        | Enum                    | Variant for the Okta loading page           |

Reviewer: ```suggestion
| `loadingPageTouchPointVariant` <ApiLifecycle access="ea" />       | Enum                    | Variant for the Okta loading page           |
```

| `errorPageTouchPointVariant`          | yes                   | yes                        | `BACKGROUND_IMAGE`              |
| `endUserDashboardTouchPointVariant`   | no                    | n/a                        | `OKTA_DEFAULT`                  |
| `endUserDashboardTouchPointVariant`   | yes                   | n/a                        | `LOGO_ON_FULL_WHITE_BACKGROUND` |

> | `loadingPageTouchPointVariant`        | n/a                   | n/a                        | `OKTA_DEFAULT` or `NONE` |

Reviewer: ```suggestion
| `loadingPageTouchPointVariant` <ApiLifecycle access="ea" />       | n/a                   | n/a                        | `OKTA_DEFAULT` or `NONE` |
```

> **Note:**
> For existing orgs, the `loadingPageTouchPointVariant` property is initialized to `OKTA_DEFAULT` if the **Okta Interstitial

> > Page** setting under **Customizations** > **Other** is enabled in the Admin Console. Otherwise, this property is initialized to `NONE`.

Reviewer: ```suggestion
> Page** setting under **Customizations** > **Other** is enabled in the Admin Console. Otherwise, this property is initialized to `NONE`. <ApiLifecycle access="ea" />
```
